widgetssdk/src/main/java/com/glia/widgets/webbrowser/WebBrowserActivity.kt

```diff
@@ -7,7 +7,14 @@ import android.os.Bundle
 import com.glia.widgets.base.FadeTransitionActivity
 import com.glia.widgets.databinding.WebBrowserAvtivityBinding
 
-
+/**
+ * Glia internal class.
+ *
+ * Will be automatically added to integrator Manifest by Manifest merger during compilation.
+ *
+ * This activity is used to display simple web pages. For example, used for displaying client's Terms and Conditions or
```

Comment:
```suggestion
 * This activity is used to display simple web pages. For example, it can be used to display links to the client's Terms and Conditions or
```